custom/icds_reports/tasks.py

```diff
@@ -1313,7 +1317,7 @@ def collect_inactive_awws():
 
     celery_task_logger.info("Creating csv file")
     export_file = BytesIO()
-    export_from_tables([['inactive AWWSs', rows]], export_file, 'csv')
+    export_from_tables([[f"inactive AWWSs {date.today().strftime('%Y-%m-%d')}", rows]], export_file, 'csv')
```

Comment:
nit: should this be today or a day before for which the inactive awws are fetched?

Reply:
keeping it the same as mentioned in the file name above.

Reply:
Ah, sounds good.